### ISSUE TITLE
Set seed for run

### ISSUE THREAD

**Issue by mfeurer** [openml/openml-python#169]
First try to do this: https://github.com/openml/openml-python/pull/133/commits/23de78b41909c0cafda2c1e098f41dbd675bd3ef

This commit won't be merged and should be redone.

**Comment by janvanrijn** [comment]
I authored a huuge pull request. 

Summarizing the state:
- Users can upload unseeded flows (a random seed will be selected and uploaded to the server)
- Users can upload flows seeded with a int (will be uploaded to the server)
- Flows with a RandomState object are rejected (too hard to reproduce)

Having this functionality, we can also re-instantiate serialized flows and parameter settings on the server. I added this functionality, so we can test the seed function (and use run instantiation). Will add some more tests tomorrow and then it will be done, I suppose. 

Furthermore, would be cool to have a feature to reinstantiate configurations based on parameter trace. Need to open an issue for this to review the state of this.
